utils/src/main/java/com/cloud/utils/UriUtils.java

```diff
@@ -200,37 +198,27 @@ public static String getUpdateUri(String url, boolean encrypt) {
     }
 
     // Get the size of a file from URL response header.
-    public static Long getRemoteSize(String url) {
-        Long remoteSize = (long)0;
+    public static long getRemoteSize(String url) {
```

Comment:
@remibergsma if you look at all the calls to getRemoteSize, they all expect a `long`. Moreover the result is neither `null` but a long, 0 or an exception, so it's best to return a `long` directly IMO.